examples/inaccessible.py

```diff
@@ -0,0 +1,32 @@
+import graphene
+
+from graphene_federation import inaccessible, external, provides, key, override
+
+from graphene_federation import build_schema
+
+
+@key(fields="x")
+class Position(graphene.ObjectType):
+    x = graphene.Int(required=True)
+    y = external(graphene.Int(required=True))
+    z = inaccessible(graphene.Int(required=True))
+    a = provides(graphene.Int(required=True), fields="x")
```

Comment:
btw, I've noticed if you do `provides(graphene.Int, fields="x")` it won't fail, but the field will not be in the schema, maybe we should check for this potential error?

Reply:
@patrick91 Upon checking, I found that this is the default behavior shown by graphene. Fields with invalid types are ignored from the schema. Adding graphene.Int() or graphene.Int(required=True) is needed for the field to be in the schema.